FreeRTOS/FreeRTOS-Kernel
# Added GCC port files for AVR Mega0 and AVR Dx.

Portable files for AVR Mega0 and AVR Dx families using GCC compiler

Description
-----------
Mainly implements the context switching and tick generation. Several tick sources are available.

Test Steps
-----------
Use the related Atmel Studio projects that will be added to the FreeRTOS repo by another PR. These projects include a blinky example and a "minimal" and "full" test scenarios


By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
